python/pyspark/sql/catalog.py

```diff
@@ -251,19 +329,31 @@ def getTable(self, tableName: str) -> Table:
         Parameters
         ----------
         tableName : str
-                    name of the table to check existence.
+            name of the table to get.
```

Comment:
add
```
            .. versionchanged:: 3.4.0
               Allow `tableName` to be qualified with catalog name.
```
?

Reply:
It's a new API in 3.4.0 so we won't have to add that log